### ISSUE TITLE
Postgres dump local restoration command shows error about `nmdc_reader` role

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/nmdc-server#2059]
As described in commit https://github.com/microbiomedata/infra-admin/commit/2b103010641794c6764ce7997f2721b7eeadd634 in `infra-admin`, @eecavanna created a new role in the production Postgres database, named `nmdc_reader`. This was to give team members read-only access to the Postgres database.

As a result, when `nmdc-server` developers try to restore production dumps into their local environments, the restore program shows the following error:

```sql
pg_restore: from TOC entry 4079; 0 0 ACL SCHEMA public postgres
pg_restore: error: could not execute query: ERROR:  role "nmdc_reader" does not exist
Command was: REVOKE USAGE ON SCHEMA public FROM PUBLIC;
GRANT ALL ON SCHEMA public TO PUBLIC;
GRANT USAGE ON SCHEMA public TO nmdc_reader;
```

This ticket represents the task of making it so the restore program does not show that error; i.e. so that that role exists in local environments before the loader script "tries to use it."
